features/step_definitions/basic_steps.rb

```diff
@@ -12,6 +12,10 @@
 
   index = ordinal ? [0, 1, 2, 3, 4].send(ordinal.lstrip) : 0
 
+  if Capybara.current_driver == (:selenium || :selenium_browser)
```

Comment:
This is going to run on the vast majority of the scenarios (if not all of them), but it only needs to be run on a few.
Use a more precise way of determining which scenarios actually need this.  I'd recommend using a hook (`@modal` or something).

Reply:
I don't know of any way to access vars, set in a hook, in a step.  Perhaps with `World` or global vars(?).  Anyway, that's messy.

I added another step `I want to create a new company` to be called before invoking the modal.

Reply:
As we talked about, let's move forward on this.  You and I can discuss this elsewhere and/or in another PR.